src/models/zener.jl

```diff
@@ -109,17 +109,17 @@ SLS_Zener = RheoModelClass(
               end,
           # Storage modulus
           Gp = quote
-                  τ = η/kᵦ
-                  denominator = 1 + τ^2*ω^2
-                  numerator = ω^2*τ^2*kᵦ
-                  numerator/denominator + kᵧ
+#                  τ = η/kᵦ
+#                  denominator = 1 + τ^2*ω^2
+#                  numerator = ω^2*τ^2*kᵦ
```

Comment:
The @. macro doesn't work on multiline expressions, so yes, probably ~5% performance difference I'd say, and less allocations.

Reply:
Fair enough. I guess, we could use @. for each line separately though it would still use more allocations. It doesn't matter as much now because the moduli we want in RHEOS are defined, and have some level of testing, but the separation was useful visually to help avoid typos in complicated expressions.